src/vswhere.lib/Console.cpp

```diff
@@ -66,7 +66,7 @@ void __cdecl Console::Write(_In_ const std::wstring& value)
     Write(value.c_str(), NULL);
 }
 
-void __cdecl Console::WriteLine(_In_ LPCWSTR wzFormat, ...)
+void __cdecl Console::WriteLine(_In_opt_ LPCWSTR wzFormat, ...)
```

Comment:
Except it's not optional.

Reply:
In this case, you're right and I wasn't thinking that the body even accepts NULL, though I can't remember why.